path_roles.go

```diff
@@ -247,5 +268,6 @@ To configure the organization
 token, set the organization field.
 `
```

Comment:
Should we describe how to set a user token here?

Reply:
Probably so (again)! I updated this in fe151be, please let me know what you think